.travis/config.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
This changes looks wrong. config.sh is not an executable AFAIU.

Reply:
> This changes looks wrong. config.sh is not an executable AFAIU.

shellcheck flags this - and the way shellcheck suppressions work it would be difficult to suppress this one.  I suppose I could add it to the global ignore list, but then we would have to ensure that a real executable script has the `#!/bin/bash`

Reply:
What is the difficulty here? Does adding the "disable ..." comment instead of the shebang not help, here?

Reply:
it should be ok in this case - but in general, if the shellcheck directive is the first line in the file, or just after the shebang, it applies to the entire file - in this case, that is the desired behavior

Reply:
fixed in https://github.com/linux-system-roles/network/pull/261/commits/e2c5cd73388c6d75f3376f44aa5f2c799199c335